public/javascripts/SVLabel/src/SVLabel/onboarding/Onboarding.js

```diff
@@ -475,7 +475,15 @@ function Onboarding(svl, actionStack, audioEffect, compass, form, handAnimation,
         var mission = missions[0];
 
         missionContainer.setCurrentMission(mission);
-        modalMission.setMissionMessage(mission, neighborhood);
+        if (missionContainer.isTheFirstMission() && user.getProperty("username") === "anonymous") {
```

Comment:
I changed this section of code in my latest commit to handle both registered and anonymous users.